app/root.tsx

```diff
@@ -55,8 +55,12 @@ export const links: LinksFunction = () => {
 		},
 		{ rel: 'manifest', href: '/site.webmanifest' },
 		{ rel: 'icon', href: '/favicon.ico' },
+		// Preload CSS as a resource to avoid render blocking
+		{ rel: 'preload', href: fontStylestylesheetUrl, as: 'style' },
 		{ rel: 'stylesheet', href: fontStylestylesheetUrl },
+		{ rel: 'preload', href: tailwindStylesheetUrl, as: 'style' },
 		{ rel: 'stylesheet', href: tailwindStylesheetUrl },
+		cssBundleHref ? { rel: 'preload', href: cssBundleHref, as: 'style' } : null,
```

Comment:
Really doesn't feel like this should be necessary, or if it should, I would expect it to be better if we moved all preload tags to before the stylesheets come in?

Could you show some before/after metrics?